src/meshapi/admin/install.py

```diff
@@ -9,6 +10,9 @@
 from import_export.admin import ExportActionMixin, ImportExportModelAdmin
 
 from meshapi.models import Install
+from meshapi.widgets import ExternalHyperlinkWidget
+
+OSTICKET_URL = os.environ.get("OSTICKET_URL", "https://support.nycmesh.net")
```

Comment:
Please add this to `.env.sample` (and the various IAC bits. @james-otten made an example here #476)

Reply:
It looks like this is already added in prod.